app.js

```diff
@@ -1,80 +1,90 @@
+// create a variable called addItem that takes "state" and "item" as arguments;
+var addQuestion = function(state, questions) {
+    // we'll want to push our item in our ITEMS array
+    // state.questions.push(question)
+    state.questions = questions
+};
+// we create a constructor function for our Questions
+var Question = function(pic, choices, correct) {
+    this.pic = pic;
+    this.choices = choices;
+    this.correct = correct;
+};
+// create a variable called questions which has an array of objects where we use Question constructor
+var questions = [
+    new Question("images/vader.jpeg", ["Kylie Jenner", "Anakin Skywalker", "Tony Montana", "Kanye West"], 1),
+    new Question("images/ironman.jpeg", ["Rick Ross", "Stevie Wonder", "Kanye West", "Iron Man"], 3),
+    new Question("images/mario.png", ["Mario", "Kanye West", "Tyga", "John Hancock"], 0),
+    new Question("images/cloud.jpg", ["Kanye West", "John Travolta", "Gerarld Butler", "Cloud"], 3),
+    new Question("images/rick.jpg", ["Stone Cold Steve Austin", "Rick & Morty", "Tom & Jerry", "Kanye West"], 1),
+    new Question("images/stewie.jpg", ["Lady Gaga", "Kanye West", "Johnny Depp", "Stewie Griffin"], 3),
+    new Question("images/bender.jpg", ["Kanye West", "Captain America", "Ash", "Bender"], 3),
+    new Question("images/goku.jpg", ["Mr. Miyagi", "Empress Bukake", "Son Goku", "Kanye West"], 2),
+    new Question("images/pikachu.png", ["Satan", "Kanye West", "Hypnotoad", "Pikachu"], 3),
+    new Question("images/kanye.jpg", ["Mr. West", "Kanye West", "Yeezy", "Pablo"], 1)
+];
 
-    $('#submit').on('click', function() {
-        checkAnswer();
-         currentQuestion++;
-        nextQuestion();
-       
-    })
+var quiz = addQuestion(state, questions);
 
-    function checkAnswer() {
-        var answer = $("input[type='radio']:checked").val();
-        if (answer == questions[currentQuestion].correct) {
-            console.log("You got it!");
-            numberCorrect++;
-        } else {
-            console.log("Incorrect!");
 
-        }
-    };
+// FUNCTION THAT RENDERS THE STATE
+var renderImage = function(state, element) {
 
-    function nextQuestion() {
-        if (currentQuestion < 10) {
-            $('.answer0').text(questions[currentQuestion].choices[0]);
-            $('.answer1').text(questions[currentQuestion].choices[1]);
-            $('.answer2').text(questions[currentQuestion].choices[2]);
-            $('.answer3').text(questions[currentQuestion].choices[3]);
-            $('.pic').attr('src', questions[currentQuestion].pic);
-        } else if(currentQuestion = 10){
-            alert("You got " + numberCorrect +" correct out of 10 questions!");
-        }
+    var result = $('.quiz-template').clone();
+    var imageElement = result.find('img');
+    imageElement.attr('src', state.questions[state.position].pic);
+    element.html(imageElement);
+}
+
+var renderChoices = function(state, element) {
+    var html = "";
```

Comment:
Could use single quotes instead of double quotes here to be consistent with how you're using single quotes in other places, e.g., line 59. Then use double quotes in your HTML markup.